ansible-scylla-node/tasks/main.yml

```diff
@@ -8,6 +8,52 @@
   package_facts:
     manager: auto
 
+- name: Set Scylla package prefix as OSS
+  set_fact:
+    scylla_package_prefix: "scylla"
+  when: scylla_edition == 'oss'
+
+- name: Set Scylla package prefix as Enterprise
+  set_fact:
+    scylla_package_prefix: "scylla-enterprise"
+  when: scylla_edition == 'enterprise'
+
+- name: Check if {{ scylla_package_prefix }} meta-package is installed
+  set_fact:
+    scylla_is_installed: "{{ true if scylla_package_prefix in ansible_facts.packages else false }}"
+
+- name: Get installed scylla version
+  set_fact:
+    installed_scylla_version: "{{ ansible_facts.packages[scylla_package_prefix][0]['version'] }}"
+  when: scylla_is_installed
+
+- name: Initialize scylla_version_to_install to {{ scylla_version }}
+  set_fact:
+    scylla_version_to_install: "{{ scylla_version }}"
```

Comment:
won't this conflict with update job? ( I forgot the flow, but if this is inited before update job it will take over variable names in case they are the same here and there)

Reply:
This fact is internal and if any other flow uses it it will overwrite it to whatever it needs.